src/app/loans/common-resolvers/loan-transaction-writeoff.resolver.ts

```diff
@@ -0,0 +1,31 @@
+/** Angular Imports */
```

Comment:
@BLasan Please use the common resolver `loan-action-button.resolver.ts` for all the actions with the 'if condition'.